### DISCUSSION TITLE
Graphic Exapansions

### DISCUSSION THREAD

**Discussion by CJRAWR** [MattFiler/LegendPlugin#172] [Ideas]
Since this is a DX11 Game that's Deferred Render, does the scope of this project include trying to make this game look nicer either through replacement of individual shaders or even wholesale render replacement such like RTX remix where they keep the game logic but take the game's outputting buffers and rerender the game? I think this game needs actual real time reflections to look its best! Thx for whatever answer

**Comment by MattFiler** [comment]
I'm currently working on reverse engineering the shader/material system in the CathodeLib project which powers the OpenCAGE tools, so some of that work may be useful for implementing any custom shaders. The same goes for custom models and textures. That'll all come to OpenCAGE's asset editing tools when it's ready.

Something on the scale of RTX Remix is waaaay out of scope for OpenCAGE as that project is all about hooking the renderer at runtime, whereas OpenCAGE is more focussed on harnessing the power of Cathode, the game's engine. It's possible that NVidia will bring Remix to DX11 at some point in the future though.

**Reply by CJRAWR** [reply]
Thank you for the quick reply, I suppose I might have made my question too big, I guess I mean to ask, the game has flat global illumination, it looks really good to the tons of area lights, but none of them cast shadows, I would love to see focus on to that! Thank you for doing work on this game!
